Source/MediaInfo/Audio/File_Iamf.cpp

```diff
@@ -37,6 +37,14 @@ namespace Elements
     const int32u iamf = 0x69616D66;
 }
 
+namespace Codecs
```

Comment:
```suggestion
namespace CodecIDs
```

CoDec = CoderDecoder, not the case here :-p.